.github/workflows/build.yml

```diff
@@ -137,6 +137,13 @@ jobs:
                 -X 'main.algoliaAppID=${{ secrets.ALGOLIA_APP_ID }}'
                 -X 'main.algoliaAPIKey=${{ secrets.ALGOLIA_API_KEY }}'
               "
+            cp LICENSE "$TARGET_DIR/LICENSE.txt"
+      - name: Copy license file to config_dir # for Linux packages
+        if: ${{ matrix.goos == 'linux' }}
+        env:
+          LICENSE_DIR: ".release/linux/package/usr/share/doc/${{ env.PKG_NAME }}"
+        run: |
+          mkdir -p "$LICENSE_DIR" && cp LICENSE "$LICENSE_DIR/LICENSE.txt"
```

Comment:
Why only for linux packages?

Reply:
This is an additional step for Linux to follow their conventions for placing a license file in the doc directory. There is also a copy step for the zip archives in line 140.